wallet/src/dapp-interface/DAppInterface.ts

```diff
@@ -86,11 +86,21 @@ export class DAppInterface {
         );
     }
 
-    public executeMoveCall(transaction: MoveCallTransaction) {
+    public executeMoveCall(tx: MoveCallTransaction) {
```

Comment:
Why did we rename this? Leaving it as `transaction` we can avoid changing 

```
{
                type: 'execute-transaction-request',
                transaction,
}
```
to

```
{
                type: 'execute-transaction-request',
                transaction: tx,
}
```